google_looker_instance/looker_instance_enterprise_full/main.tf

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START looker_google_looker_instance_enterprise_full]
+# Creates an Enterprise edition Looker (Google Cloud core) instance with full, Private IP functionality.
+resource "google_looker_instance" "main" {
+  name               = "my-instance"
+  platform_edition   = "LOOKER_CORE_ENTERPRISE_ANNUAL"
+  region             = "us-central1"
+  private_ip_enabled = true
+  public_ip_enabled  = false
+  reserved_range     = google_compute_global_address.looker_range.name
+  consumer_network   = data.google_compute_network.looker_network.id
+  admin_settings {
+    allowed_email_domains = ["google.com"]
+  }
+  encryption_config {
+    kms_key_name = "looker-kms-key"
+  }
+  maintenance_window {
+    day_of_week = "THURSDAY"
+    start_time {
+      hours   = 22
+      minutes = 0
+      seconds = 0
+      nanos   = 0
+    }
+  }
+  deny_maintenance_period {
+    start_date {
+      year  = 2050
+      month = 1
+      day   = 1
+    }
+    end_date {
+      year  = 2050
+      month = 2
+      day   = 1
+    }
+    time {
+      hours   = 10
+      minutes = 0
+      seconds = 0
+      nanos   = 0
+    }
+  }
+  oauth_config {
+    client_id     = "my-client-id"
+    client_secret = "my-client-secret"
+  }
+  depends_on = [
+    google_service_networking_connection.looker_vpc_connection
+  ]
+}
+
+resource "google_service_networking_connection" "main" {
+  network                 = data.google_compute_network.looker_network.id
+  service                 = "servicenetworking.googleapis.com"
+  reserved_peering_ranges = [google_compute_global_address.looker_range.name]
+}
+
+resource "google_compute_global_address" "main" {
+  name          = "looker-range"
+  purpose       = "VPC_PEERING"
+  address_type  = "INTERNAL"
+  prefix_length = 20
+  network       = data.google_compute_network.looker_network.id
+}
+
+data "google_project" "main" {}
+
+data "google_compute_network" "main" {
+  name = "looker-network"
+}
+
+resource "google_kms_crypto_key_iam_member" "main" {
+  crypto_key_id = "looker-kms-key"
```

Comment:
I get an error on this resource:
```
│ Error: Error parsing resource ID for looker-kms-key: Invalid CryptoKey id format, expecting `{projectId}/{locationId}/{KeyringName}/{cryptoKeyName}` or `{locationId}/{keyRingName}/{cryptoKeyName}, got id: looker-kms-key`
│ 
│   with google_kms_crypto_key_iam_member.main,
│   on main.tf line 89, in resource "google_kms_crypto_key_iam_member" "main":
│   89: resource "google_kms_crypto_key_iam_member" "main" {
│ 
╵
```

Could be resolved by creating the resource for `crypto_key_id` and including it in the sample

Reply:
Done.